Include/internal/pycore_runtime.h

```diff
@@ -192,6 +236,14 @@ typedef struct _Py_DebugOffsets {
         uint64_t size;
         uint64_t collecting;
     } gc;
+
+    struct _gen_object {
+        uint64_t size;
+        uint64_t gi_name;
+        uint64_t gi_iframe;
```

Comment:
This is an implementation detail, and may not exist in future versions of CPython

Reply:
Yeah, see also my comment [here](https://github.com/python/cpython/pull/124640#discussion_r1805079790). I don't think it's worth anyones time to debate these offsets. Yes they are subject to break and that's fine, they are not a public API. Profilers and debuggers expect this kind of stuff to drift.

Reply:
OK. I worry that when they do break, we'll be pressured to restore whatever field they were relying on.

I'd be happier if these offsets were in their own header, with a large clear warning about breakage.

Reply:
I don't think which header file they live in should matter, we don't need a new header for this specific internal implementation detail, this already is such a header.  Lets just add the desired code comment about stability expectations and breakage as instructions to our future selves that if these need to change, so be it, and that the internal implementation detail tests relying on these can be marked as expected failures at that time.